gravatar-enhanced.php

```diff
@@ -4,15 +4,15 @@
 Plugin URI: https://wordpress.org/extend/plugins/gravatar-enhanced/
 Description: Enhanced functionality for Gravatar-ifying your WordPress site. Once you've enabled the plugin, go to the "Avatars" section on the <a href="options-discussion.php">Discussion Settings page</a> to get started.
 Author: Automattic
-Version: 0.6.0
+Version: 0.6.1
```

Comment:
IMO I would go with 0.7.0, this is not a "fix" but a proper update (of the dependency at least).

Also I would add a mention to the new hovercard design (more vertical?). In the changelog.

Reply:
I would think of this as a patch.
But I'm with changing the version to `0.7.0`.

I'm not sure I would mention the hovercard design change.
That feels like something that should live in the Hovercards library changelog.